tests/phpunit/src/Commands/Push/PushArtifactCommandTest.php

```diff
@@ -67,6 +53,67 @@ public function testPushArtifact(): void {
     $this->assertStringContainsString('[0] Sample application 1', $output);
     $this->assertStringContainsString('Choose a Cloud Platform environment', $output);
     $this->assertStringContainsString('[0] Dev, dev (vcs: master)', $output);
+    $this->assertStringContainsString('Pushing changes to Acquia Git (site@svn-3.hosted.acquia-sites.com:site.git)', $output);
+  }
+
+  public function testPushArtifactWithAcquiaCliFile() {
+    $this->datastoreAcli->set('push.artifact.destination-git-urls', [
+      'https://github.com/example1/cli.git',
```

Comment:
This doesn't match the fixture file, which makes me wonder are we even using the fixture file? Can it be removed from the repo?